### ISSUE TITLE
Remove redirect html tag

### ISSUE THREAD

**Issue by kelshmo** [Sage-Bionetworks/synapseDocs#605]
@kdaily mentioned some pages have a redirect tag that causes url links to be opened in a new browser page. 

These should be removed. Instead, each viewer's browser settings should determine how pages are opened.

**Comment by dilekuzulmez** [comment]
I would like to help you. But which links they are ?

**Comment by kdaily** [comment]
Hi @dilekuzulmez, welcome!

In the branch [`gh-pages`](https://github.com/Sage-Bionetworks/synapseDocs/tree/gh-pages) there are Markdown links that have a `{:target="_blank"}` tag after them. They look like:

```
[Some link text](https://www..../){:target="_blank"}
```

We need to remove `{:target="_blank"}`:

```
[Some link text](https://www..../)
```

If you would like to make a contribution, please [fork this repository](https://help.github.com/en/articles/fork-a-repo) and [make a new branch](https://help.github.com/en/articles/creating-and-deleting-branches-within-your-repository) with your changes. Open a pull request against the `gh-pages` branch and someone can review your changes.

Thank you!

**Comment by kdaily** [comment]
Closed by #607 and #608.
